e2e2/test/cases/training/manifests/bert-training.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: kubeflow.org/v2beta1
+kind: MPIJob
+metadata:
+  name: bert-training
+spec:
+  slotsPerWorker: 8
+  runPolicy:
+    backoffLimit: 20
+    cleanPodPolicy: Running
+  mpiReplicaSpecs:
+    Launcher:
+      replicas: 1
+      template:
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - image: {{.BertTrainingImage}}
+            imagePullPolicy: Always
+            name: bert-training
+            env:
+            - name: NCCL_DEBUG
+              value: "TRACE"
+            - name: MASTER_ADDR
+              value: "bert-training"
+            - name: MASTER_PORT
+              value: "12355"
+            command:
+            - /opt/amazon/openmpi/bin/mpirun
+            - --allow-run-as-root
+            - --tag-output
+            - -np
+            - "32"
+            - -bind-to
+            - none
+            - -map-by
+            - slot
+            - -x
+            - PATH
+            - -x
+            - LD_LIBRARY_PATH
+            - -x
+            - NCCL_DEBUG
+            - -x
+            - MASTER_ADDR
+            - -x
+            - MASTER_PORT
+            - --mca 
+            - pml
+            - "^cm"
+            - --mca
+            - routed
+            - direct
+            - --oversubscribe
+            - --mca
+            - orte_base_help_aggregate 
+            - "0"
+            - python
+            - train.py
+    Worker:
+      replicas: 4
+      template:
+        spec:
+          volumes:
+          - name: dshm
+            emptyDir:
+              medium: Memory
+          containers:
+          - image: {{.BertTrainingImage}}
+            imagePullPolicy: Always
+            name: bert-training-worker
+            volumeMounts:
+            - mountPath: /dev/shm
+              name: dshm
+            resources:
+              requests:
+                nvidia.com/gpu: 8
+                vpc.amazonaws.com/efa: 0
+              limits:
+                nvidia.com/gpu: 8
+                vpc.amazonaws.com/efa: 0
```

Comment:
I think we shouldn't hard code this, since we might need to run it in different node configurations (e.g. node type, node count). 
Check here for a reference on how not to hardcode this.
https://github.com/aws/aws-k8s-tester/blob/main/e2e2/test/cases/nvidia/main_test.go#L98-L144
https://github.com/aws/aws-k8s-tester/blob/main/e2e2/test/cases/nvidia/manifests/mpi-job-nccl-test-multi-node.yaml

Reply:
Sure, we can parameterize it for future proofing. Right now all tests will be ran on an instance with 8 NVIDIA GPUs, but I have no problem with this. Will make the update.